packages/eas-cli/src/credentials/ios/utils/authType.ts

```diff
@@ -1,6 +1,7 @@
 import { RequestContext } from '@expo/apple-utils';
 
 /** Is the request context App Store Connect only with no access to cookies authentication. */
+// todo: perhaps good to differentiate jwt vs cookies authentication upstream?
 export function isAppStoreConnectTokenOnlyContext(authContext: RequestContext): boolean {
```

Comment:
Internally we allow for both types and defer to using App Store Connect API when available [ref](https://github.com/expo/third-party/blob/3ac7f272d26bfc47e7660c0bb83b4bdf7cf6520d/packages/app-store/src/network/Request.ts#L107-L119). I added this extra step so `eas-cli` had explicit control on how we use the API. This is largely due to the fact that `apple-utils` doesn't have a proper governing body (or even CHANGELOG) to prevent logic changes like flipping the authentication priority to favor cookies first.

The purpose of this specific function is currently to bail out of API calls that we know do not work (like push keys, capability ids, provisioning profile repair).